cmd/routers.go

```diff
@@ -38,6 +38,8 @@ func registerDistErasureRouters(router *mux.Router, endpointServerPools Endpoint
 	// Register distributed namespace lock routers.
 	registerLockRESTHandlers()
 
+	// TODO: We probably don't need OPTIONS preflight matching on this router, but just check.
```

Comment:
Resolve.

Reply:
This TODO is a question for reviewers: Do we need OPTIONS on these? I doubt it because these are `registerDistErasureRouters` which seem to all deal with peer-to-peer endpoints (grid?). I doubt that a browser is using fetch() under cross-origin against one of these APIs?

Reply:
no they won't be, in-fact we should reject all CORS, OPTIONs request at `/minio/` resource entirely.

Reply:
@harshavardhana Don't we need to support a use case where someone makes browser-based requests to the MinIO admin API? 

E.g. if a user has a custom web UI that is cross-domain and does some direct requests to MinIO (not using / bypassing MinIO console)